README.md

```diff
@@ -54,9 +56,13 @@ and install it in development mode:
 
 ```shell
 git clone https://github.com/etalab/udata-gouvfr.git
-pip install -e udata-gouvfr
+cd udata-gouvfr
+pre-commit install
+pip install -e . -r requirements/test.pip -r requirements/develop.pip
```

Comment:
You should also install `install.pip`, right?

Reply:
`-e .` uses `install_requires` from `setup.py` which gets its content from `install.pip`, so we shouldn't need to.

Reply:
(spending two weeks on packaging has cleared my mind a lot ahah)

Reply:
![](https://media.giphy.com/media/a47CWNuZokDXwy7kjr/giphy.gif?cid=ecf05e47wn9ul6e2h1a2pi8sns586on28cax1j11gsp1xbcm&rid=giphy.gif&ct=g)